web/src/title/TitleMode.svelte

```diff
@@ -204,4 +226,8 @@
   .project-list li {
     padding-left: 1em;
   }
+
+  .project-name {
+    overflow-wrap: anywhere;
```

Comment:
![image](https://github.com/user-attachments/assets/0c1c4da6-c503-499a-9389-aab00d0449cc)
The wrapping looks reasonable to me, keeps the buttons always the same size/location